Formula/n/nowplaying-cli.rb

```diff
@@ -0,0 +1,18 @@
+class NowplayingCli < Formula
+  desc "Retrieves currently playing media, and simulates media actions"
+  homepage "https://github.com/kirtan-shah/nowplaying-cli"
+  url "https://github.com/kirtan-shah/nowplaying-cli/archive/refs/tags/v1.2.1.tar.gz"
+  sha256 "bb49123c66282b6495c245589313afc94875a7b0e82c9ae9f79d6f25e7503db4"
+  license "GPL-3.0-or-later"
+
+  depends_on :macos
+
+  def install
+    system "make"
+    bin.install "nowplaying-cli"
+  end
+
+  test do
+    system "#{bin}/nowplaying-cli", "get-raw"
```

Comment:
We can. The application returns non-zero on errors (such as the inability to read data from the API) and returns success (0) when the data was read successfully. This could be "(null)" when no information is available, or a macOS dictionary if data is populated via MediaRemote.

An assertion runs the risk of changes to output in subsequent versions, but I guess this is tracked against subsequent PRs to support newer versions. I expect by then we'll have a test framework that would provide more concrete results of testing.